geomesa-raster/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,25 @@
+#
+#  Copyright 2014 Commonwealth Computer Research, Inc.
```

Comment:
I forget where we left this...  What's the right year (or range of years) to use in new files?